.github/workflows/stale.yml

```diff
@@ -36,3 +36,7 @@ jobs:
         stale-pr-label: 'no-pr-activity'
         exempt-pr-labels: 'awaiting-approval,work-in-progress'
         only-labels: 'awaiting-feedback,awaiting-answers'
+        days-before-pr-stale: '10'
+        days-before-pr-close: '20'
+        days-before-issue-stale: '60'
```

Comment:
what happens when something becomes stale? Is the issue labeled?

Most issues have no updates in practice, right? I'm worried that issues labeled as stale may deter external contributors from working on them.

On the other hand, having a stale label would help us decide whether we should close them manually or not, which seems like a good idea. How do we remove the stale issue? Manually? Or we're supposed to add a comment to restart the 60 day period?

Reply:
`If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart` also we can have exempt-labels to avoid marking an issue as stale.